client/src/pages/Profile.js

```diff
@@ -544,7 +485,98 @@ const Profile = ({
           <Verification gtmPrefix={GTM.user.profilePrefix} />
         )}
         <WhiteSpace />
-        <ProfileTabs tabData={isSelfViews(tabViews, isSelf)} />
+        {window.width <= parseInt(mq.phone.wide.maxWidth) ? (
+          <MobileMenuWrapper
+            defaultSelectedKeys={[sectionView]}
+            selectedKeys={sectionView}
+            onClick={handleMenuToggle}
+          >
+            {navMenu.map((item, index) => (
+              <Menu.Item key={item.name} disabled={item.disabled}>
+                {item.name}
+              </Menu.Item>
+            ))}
+          </MobileMenuWrapper>
+        ) : null}
```

Comment:
I'm a bit confused here. Can you instead of :null render desktop view?

Reply:
We can in this instance because they are going in the same place on this page. It might not be structured that way once we arrange the content for desktop view. I guess I should ask you and @aditilamkhede  if you would like me to arrange the page so there is a sidebar and you guys can just drop your content in the feed section on the right?

I can roll that up in this PR or we could do a seperate feature/desktop design PR. What makes sense to the team?